packages/airnode-node/config/config.example.json

```diff
@@ -20,7 +20,11 @@
           "value": 3.12,
           "unit": "gwei"
         },
-        "fulfillmentGasLimit": 500000
+        "fulfillmentGasLimit": 500000,
+        "withdrawalRemainder": {
+          "value": 0,
```

Comment:
Nit: Maybe add a non null value, since if the user wants to have 0 remainder he can just omit this.

Reply:
Given `withdrawalRemainder` is only relevant to Optimism at the moment and is covered in the tests, I opted for omitting it from this `config.example.json` rather than leaving it as zero (potentially confusing) or adding a non-zero value (which wouldn't be appropriate for most chains). I'll describe it in the docs and within the Chain Idiosyncrasies page ([AN-761](https://api3dao.atlassian.net/browse/AN-761)).